### PR TITLE
Document current features of Dlang-Bot

### PR DESCRIPTION
**Pull request by wilzbach**
It's an ever-growing list :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,4 +105,79 @@ automatically remove set all auto-merge labels from the PR
 _Note_: at the moment the Dlang-Bot doesn't store _any_ authentication token, so
 it can't perform actions on behalf of others.
 
-<a name="canceling-state" />
+<a name="warn-common-mistakes" />
```
Comment (CyberShadow) [inline README.md]: Doesn't GitHub automatically create linkable anchors for headings? It even shows the link thing in the margin when you hover over it.

Comment (wilzbach) [inline README.md]: > Doesn't GitHub automatically create linkable anchors for headings? It even shows the link thing in the margin when you hover over it.

It does, but it's rather annoying to keep it in sync. For example:

https://github.com/dlang-bots/dlang-bot#nerdy-details-1

The manual way here has the advantage that one doesn't break links by accident.

Comment (CyberShadow) [inline README.md]: FWIW there are Markdown TOC generators such as https://github.com/jonschlinkert/markdown-toc which reuse GitHub's existing anchors.

I remember I used one for https://github.com/CyberShadow/aconfmgr#table-of-contents, though it needed some minor tweaks.

Comment (wilzbach) [inline README.md]: Hehe thanks. I am aware of them, but the current solution works well and imho  isn't a big overhead. I would be very happy if the Dlang-Bot ever gained so many features that we need to opt for such an approach.

**Comment by wilzbach** [comment]
I am going to merge this now as it just modifies the plain text README (no harm done) and currently sadly not many people care about the bot anyways..

**Pull request merged**